src/NServiceBus.Core/UnitOfWork/UnitOfWorkBehavior.cs

```diff
@@ -42,7 +47,7 @@ class UnitOfWorkBehavior : Behavior<PhysicalMessageProcessingContext>
             }
         }
 
-        List<Exception> AppendEndExceptionsAndRethrow(Exception initialException)
+        List<Exception> AppendEndExceptionsAndRethrow(Stack<IManageUnitsOfWork> unitsOfWork, Exception initialException)
```

Comment:
there I fixed it:

![swissinq](https://cloud.githubusercontent.com/assets/3524870/11392427/f0804c28-9359-11e5-8f78-da9bc4722f3f.png)